### ISSUE TITLE
IP Anonymization in Google Analytics

### ISSUE THREAD

**Issue by kinueng** [OpenLiberty/openliberty.io#83]
Integrate IP anonymization into Google Analytics due to the May 2018 General Data Protection Regulation for the European Union.

Google article on IP anonymization in Google Analytics:
https://support.google.com/analytics/answer/2763052?hl=en&ref_topic=2919631

Google Analytics Anonymize IP API documentation:
https://developers.google.com/analytics/devguides/collection/analyticsjs/field-reference#anonymizeIp

**Comment by mchenggit** [comment]
This is needed for GDPR compliance, with due date 15 Feb, 2018

**Comment by kinueng** [comment]
In Google Tag Manager, I have added the Google Analytics `anonymizeIp : true` property.  I verified that the IP anonymization property is enabled by looking for `aip=1` in the google analytics traffic on openliberty.io.

Example of verification:
1.  Open new tab in Chrome
2.  Enable Chrome debug tools
3.  In tools, open the Network tab
4.  Go to openliberty.io
5.  Filter the network data with the word `analytics`
6.  Observe that the `collect` request has the `aip=1`, meaning IP addresses are anonymized.

> The IP anonymization parameter looks like this: &aip=1
> src: https://support.google.com/analytics/answer/2763052?hl=en&ref_topic=2919631
